lib/components/home/artist_rail.js

```diff
@@ -0,0 +1,57 @@
+/* @flow */
+'use strict';
+
+import Relay from 'react-relay';
+import React from 'react';
+import { View, Text } from 'react-native';
+
+import Spinner from '../spinner';
+
+class ArtistRail extends React.Component {
```

Comment:
Thanks for the example file! Is it ok if I merge this with the files I already have in my branch shortly?